index.html

```diff
@@ -28,32 +28,36 @@
 		<section class="hello-section">
 			<h1 class="hello-there">Hello there!</h1>
```

Comment:
this class isn't in use? is this necessary? could target it with ".hello-section h1 {}"

(just me being a pedantic/not entirely sure of best practice yet>